doc/installation.rst

```diff
@@ -1,129 +1,217 @@
+   option. This provides a much better experience than Docker on Windows.
+   Docker and Raspberry Pi installations are being deprecated—see notes below.
+
 
-Linux, MacOS, Termux and WSL2 (Preferred)
-------------------------------------------
+Linux, MacOS, and Termux (Preferred)
+------------------------------------
+
+**A native Linux installation is the most robust and recommended approach.**
 
 For all systems, make sure you have **git** and **curl** installed.
 
 * Arch-based system (like vanilla Arch or Manjaro)
 
   ``sudo pacman -S git curl``
 
-* Debian-based system (like Ubuntu or Mint) – WSL2 with Ubuntu/Debian also follows this flow:
+* Debian-based system (like Ubuntu or Mint)
+
   ``sudo apt-get install git curl``
 
 * Termux-based system (https://termux.dev) can be used on Android
+
   ``apt-get install git curl``
 
-* macOS: install homebrew (https://brew.sh/) and then run
+* MacOS install homebrew (https://brew.sh/) and then run
+
   ``brew install git curl``
 
-After you have **git** and **curl**, install IoTempower in the terminal with:
+After you have **git** and **curl**, install IoTempower in the terminal with the following:
 
 .. code-block:: bash
 
    curl -L https://now.iotempower.us | bash -
 
-Hit enter on all questions for default settings and full features (recommended).
+Hit enter on all questions to use default settings and enable full features (recommended).
 
-Convenience tools (like Midnight Commander, micro, tilde) provide easier file and text editing on the command line.
+Convenience tools are optional but we recommend them because Midnight Commander is nicer file
+manager in the command line interface and integrates nicely with IoTempower.
+Micro, and tilde are nicer/easier editors than nano or vim as they support Windows-like
+keyboard shortcuts for select, copy, paste, save, and quit.
 
-*Alternatively, install directly with default prompts:*
+*Alternatively, you can install everything directly without the prompt questions by running:*
 
 .. code-block:: bash
 
    curl -L https://now.iotempower.us | bash -s -- --default
 
-After installation, open a new terminal and run ``iot`` from anywhere. (Or, for debugging, run ``bash run`` in the iot directory.)
+After the installation is finished, open a new terminal, and just run ``iot`` from anywhere
+(you can also run ``bash run`` in the iot directory if things go wrong,
+but only for debugging purposes).
+
+After successfully entering IoTempower (the prompt
+should have changed and started now with IoT),
+start configuring your first IoT node,
+see `First IoT Node <first-node.rst>`_.
+
 
-After entering IoTempower for the first time (prompt should now indicate IoT), start configuring your first IoT node (see `First IoT Node <first-node.rst>`_).
+Windows Host with WSL2 + usbipd-win
+-----------------------------------
+
+.. note::
+   **New Option:** For Windows 10 (2004+) and Windows 11 users, WSL2 combined
```

Comment:
not important to stress that this is new and best mix into the previous section as remarked already above